### ISSUE TITLE
Yarn and Python2 missing from PATH

### ISSUE THREAD

**Issue by ghost** [andrewmackrodt/dockerfiles#4]
The recent update to the nodejs image looks to have broken access to Yarn and python2 is no longer installed

the Yarn issue is easy to get around by passing the full path but I am getting

`gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.`

when running yarn install and from what i can see python2 is not installws

**Comment by st-h** [comment]
This possibly might be related, when trying to launch chromium (only happens with recent docker image)

```
Error: Browser exited unexpectedly
1289            Non-zero exit code: 1
1290            Stderr: 
1291             
1292            Command '/usr/bin/chromium-browser' requires the chromium snap to be installed.
1293            Please install it with:
1294            
1295            snap install chromium
```

Are there any older docker images available? https://hub.docker.com/r/andrewmackrodt/nodejs-chromium/tags?page=1&ordering=last_updated only lists the latest version

**Comment by andrewmackrodt** [comment]
Sorry for the long delay in responding to issues. This likely happened between changing the base image from bionic (18.04) to focal (20.04). I've added `python3` (with a path alias of `python`) to the image.

@st-h that error came about when Ubuntu changed the `chromium-browser` package to support snap only. I updated it in January to use the package from `bionic-updates` which has remained working so far 🤞.

I've updated the Dockerfiles and pushed new images. (All images are built nightly at https://jenkins.mackrodt.io/job/dockerfiles/)
